src/Reflection/ReflectionExtension.php

```diff
@@ -109,7 +109,11 @@ public function getGlobalsSize(): int
      */
     public function getGlobals(): ?CData
     {
-        return $this->moduleEntry->globals_ptr;
+        if(ZEND_THREAD_SAFE) {
```

Comment:
https://github.com/php/php-src/blob/1c33ddb5e5598c5385c4c965992c6e031fd00dd6/Zend/zend_modules.h#L87 - not sure how to deal with this in the thread-safe case